### ISSUE TITLE
Upgrade python-scraperlib to 3.x, including CLI support for description / long_description flags

### ISSUE THREAD

**Issue by benoit74** [openzim/gutenberg#188]
See https://github.com/openzim/overview/issues/21

**Comment by Lorrainnn** [comment]
Hi! Just checking — is the task to upgrade of python-scraperlib still needed at this stage? If so, feel free to assign it to me or I could focus on other appropriate issues mentioned in issue https://github.com/openzim/overview/issues/21.
:)

**Comment by benoit74** [comment]
Yes, but you now have to upgrade to 5.x which has major breaking changes, not really straightforward. You can have a look at https://github.com/openzim/mindtouch/tree/main/scraper which is already in 5.x for inspiration.

**Comment by Lorrainnn** [comment]
Some issues with update to 5.x:
In order to use the updated function like `compute_descriptions`
The python-scraperlib after 5.x `requires-python = ">=3.12,<3.13"`
However, our current project is based on `requires-python = ">=3.11,<3.12"`

The solution could be either update general python config first or manually include functions(maybe bad ideas.

**Comment by benoit74** [comment]
You should indeed upgrade to Python 3.13 in the same run/PR, sorry for not having mentioned it. You can update other dependencies as well.

No big problem expected ... but this is usually when reality comes to bit me ^^
